src/auth/user.ts

```diff
@@ -36,6 +36,16 @@ export function useCurrentUser(name?: string) {
   return authUserMap.get(useFirebaseApp(name))!
 }
 
+/**
+ * Returns a boolean indicating if the current user has finished loading and is no longer undefined. This is useful for
+ * hiding navigation until the current user is loaded and we know if they are authenticated or not.
+ * @param name - name of the application
+ */
+export function isUserLoaded(name?: string) {
```

Comment:
Let's name this `useIsUserLoaded()`. The `use` prefix is important in this case

Reply:
You're right, it was late and I wasn't thinking. Of course it's a composable.